.github/workflows/gitleaks.yml

```diff
@@ -6,10 +6,13 @@ jobs:
   gitleaks:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: gitleaks-action with defaults
-        uses: zricethezav/gitleaks-action@master
-      - name: gitleaks-action with config
-        uses: zricethezav/gitleaks-action@master
-        with:
-          config-path: .gitleaks.yml
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: gitleaks-action with config
+      uses: zricethezav/gitleaks-action@master
+      with:
+        # test custom path for config
+        config-path: gitleaks.toml
+    - name: gitleaks-action with defaults will fail
+      uses: zricethezav/gitleaks-action@master
```

Comment:
...will fail as result of the current `entrypoint.sh` script. Why the default config path is dropped? I do not understand the rationale behind.

Reply:
One of the issues this patchset fixes is that previously the action would pass even if misconfigured.

This action deliberately demonstrates failure. It was useful to me when developing - not sure it needs to stick around.